benefits/templates/admin/agency-base.html

```diff
@@ -26,6 +26,31 @@ <h1 class="p-0 m-0 text-white">Administrator</h1>
 {% endblock branding %}
 
 {% block usertools %}
+    {% if has_permission %}
+        <div id="user-tools">
+            {% block welcome-msg %}
+                <span class="text-uppercase text-white">
+                    {% translate "Welcome," %}
+                    <strong>{% firstof user.get_short_name user.get_username %}</strong>. </span>
+            {% endblock welcome-msg %}
+            {% block userlinks %}
+                {% if user.is_active and user.is_staff %}
+                    {% url 'django-admindocs-docroot' as docsroot %}
+                    {% if docsroot %}
+                        <a href="{{ docsroot }}">{% translate "Documentation" %}</a> /
+                    {% endif %}
+                {% endif %}
+                {% if user.has_usable_password %}
+                    <a href="{% url 'admin:password_change' %}">{% translate "Change password" %}</a> /
+                {% endif %}
+                <img class="icon" width="15" height="15" src="{% static "img/icon/box-arrow-right.svg" %}" alt="" />
+                <form id="logout-form" method="post" action="{% url 'admin:logout' %}">
+                    {% csrf_token %}
+                    <button type="submit">{% translate "Sign out" %}</button>
```

Comment:
Adding a `class="border-0"` will remove the border. Then you can delete: https://github.com/cal-itp/benefits/pull/2324/files#diff-77a3765ab5e3cdb49fb5b92ca214d69e2821102f48112dc9eba1a9429ec902fcR71-R73

Reply:
Ah, that makes sense. Cleaned it up in aa73dca7d3981c5bd06504cb9aa5d23011dfc39e and 51f2462ab1d09b4a0ca1bc12e45b9f3612c45e25